docs/review/README.md

```diff
@@ -2,184 +2,158 @@
 
-You can watch the version from the right panel, enlarge it by grabbing the side of the panel, or even go fullscreen ![Fullscreen](../img/getting-started/fullscreen.png).
+You can watch the version from the right panel, enlarge it by grabbing the side of the panel, or even enter fullscreen by clicking this icon ![Fullscreen](../img/getting-started/fullscreen.png).
 
 ![review on global page](../img/getting-started/review_global.png)
 
-You can also review the 3D file (`.glb` file) as a wireframe or add a `.HDR` file to check the lighting. 
+In the same way, you can also review 3D files (`.glb` file) as a wireframe or add a `.HDR` file to check the lighting.
 
-You can move around the 3D file by dragging and dropping your cursor and zooming in or out with the scroll.
+You can move around the 3D file by dragging and dropping your cursor and zooming in or out with the scroll wheel.
 
 ![review 3d wireframe](../img/getting-started/review_wireframe.png)
 
 ![Preview background example](../img/getting-started/preview_background_example.png)
 
 See the **Customization** section for more details.
 [Pipeline Customization](../customization-pipeline/README.md)
+<!-- Dead link, need to confirm where this goes to -->
 
-You can draw directly on the preview with the **Pencil** tool and select a color ![Pencil](../img/getting-started/draw.png).
+You can select a color to use and then draw directly on the preview with the **Pencil** tool ![Pencil](../img/getting-started/draw.png).
 
 ![review drawing](../img/getting-started/review_comment.png)
 
-You can also add **text** on the frame.
+You can also annotate the frame with **text**.
 
 ![review drawing text](../img/getting-started/review_comment_text.png)
 
-
-If you need to see the preview without your comment, **hide your annotation**, you can click on the **Fountain Pen** button.
+If you'd like to temporarily **hide annotations**, click the **Fountain Pen** button.
 ![Hide Annotation](../img/getting-started/hide_annotation.png)
 
+If you would like to refer back to a specific frame in your comments, you can easily tag a frame by typing `@`, which opens a sub-menu with the team list, and then start typing **frame**.
 
-You can tag a frame (whatever version you are in), type the `@`and it opens a sub-menu with the team list, and start typing **frame**.
+This will then add a timestamp of the current frame to this comment, and clicking on that timestamp will jump to said frame.
 
 ![Tag a frame](../img/getting-started/tag_frame.png)
 
-
 You can also export your drawings on each frame by attaching the **screenshots as attachments**.
 
 Click on the **Add attachment** button.
 
 ![attachment](../img/getting-started/attachment_retake.png)
 
-You can also choose to **Attach snapshots from your annotation** on the attachment option.
+You can also choose to **Attach snapshots from your annotation** in the attachment options.
+
+<!-- This doesn't make sense, need to confirm what this is referring to -->
```

Comment:
You can add annotated frames as picture attachments of the comment.

Reply:
Thanks, this has been expanded on and updated in the latest commit